samples/sensor/max31865/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: MAX31865 temperature sensor
+tests:
+  sample.sensor.max31865:
+    harness: sensor
+    tags: samples sensors
+    depends_on: spi
+    platform_allow: particle_xenon
```

Comment:
Remove the `platform_allow` so this is not restricted to one board.

And add `filter: dt_compat_enabled("maxim,max31865")` so this can be built (via twister) any boards.